javadoc/com/stripe/android/model/Card.CardBrand.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
(Note: ignore the javadoc changes -- those are all getting removed by the other diff)